tests/acceptance/features/apiGraph/roleManagementEndpoint.feature

```diff
@@ -0,0 +1,605 @@
+Feature: verify role management functionality in the Graph API
+  As an  admin
+  I want to test the role management endpoints
+  So that I can find out if those endpoints are working correctly or not
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
+
+  Scenario: admin user gets a list of all available permission role definitions
+    Given the administrator has assigned the role "Admin" to user "Alice" using the Graph API
```

Comment:
Can we do it with normal user?
```suggestion
  Scenario: get a list of permissions role definitions
```

Reply:
No, we can't. It throws `401` for the normal user.

Reply:
Then we should add scenario for that case also

Reply:
I am sorry that I tested the wrong scenario where it gave 401 for normal user. 
But here you  are correct, i.e, it can be done with normal user too. Corrected the scenarios. Thanks :grin: